packages/vector_graphics_compiler/bin/vector_graphics_compiler.dart

```diff
@@ -41,10 +43,16 @@ final ArgParser argParser = ArgParser()
     help: 'Allows for overdraw optimizer to be enabled or disabled',
     defaultsTo: true,
   )
+  ..addOption(
+    'input-dir',
+    help: 'The path to a directory containing one or more SVGs. '
+        'Only includes files that end with .svg. '
+        'Cannot be combined with --input or --output.',
+  )
```

Comment:
I have a slight preference for having an input file that contains paths and what to name them, which is probably faster in network file share based environments and doesn't involve any strangeness around directory traversal/file name pattern matching, but this is probably fine too.

Reply:
we could make input a multi-option too. For local development pointing at a directory is very easy though

Reply:
Multi-option runs into issues on Windows where there are limitations on commandline length

Reply:
yeah, also matching up with outputs is kind of weird.

Reply:
I don't have a usecase for this right now, because as far as I know how to write the google3 integration it will still be done an input at a time. Can we punt on this?

Reply:
Yup